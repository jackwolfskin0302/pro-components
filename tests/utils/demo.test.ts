import demoTest from '../demo';

demoTest('utils');
