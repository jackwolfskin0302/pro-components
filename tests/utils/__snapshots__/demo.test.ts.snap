// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`utils demos renders ./packages/utils/src/demos/subTitle.tsx correctly 1`] = `
Array [
  <div
    class="ant-space ant-space-horizontal ant-space-align-center ant-pro-core-label-tip"
  >
    <div
      class="ant-space-item"
      style="margin-right: 4px;"
    >
      标题
    </div>
    <div
      class="ant-space-item"
      style="margin-right: 4px;"
    >
      <div
        class="ant-pro-core-label-tip-subtitle"
      >
        <span>
          副标题
        </span>
      </div>
    </div>
    <div
      class="ant-space-item"
    >
      <span
        aria-label="info-circle"
        class="anticon anticon-info-circle ant-pro-core-label-tip-icon"
        role="img"
      >
        <svg
          aria-hidden="true"
          class=""
          data-icon="info-circle"
          fill="currentColor"
          focusable="false"
          height="1em"
          viewBox="64 64 896 896"
          width="1em"
        >
          <path
            d="M512 64C264.6 64 64 264.6 64 512s200.6 448 448 448 448-200.6 448-448S759.4 64 512 64zm0 820c-205.4 0-372-166.6-372-372s166.6-372 372-372 372 166.6 372 372-166.6 372-372 372z"
          />
          <path
            d="M464 336a48 48 0 1096 0 48 48 0 10-96 0zm72 112h-48c-4.4 0-8 3.6-8 8v272c0 4.4 3.6 8 8 8h48c4.4 0 8-3.6 8-8V456c0-4.4-3.6-8-8-8z"
          />
        </svg>
      </span>
    </div>
  </div>,
  <div
    class="ant-space ant-space-horizontal ant-space-align-center ant-pro-core-label-tip"
  >
    <div
      class="ant-space-item"
      style="margin-right: 4px;"
    >
      标题
    </div>
    <div
      class="ant-space-item"
      style="margin-right: 4px;"
    >
      <div
        class="ant-pro-core-label-tip-subtitle"
      >
        副标题
      </div>
    </div>
    <div
      class="ant-space-item"
    >
      <span
        aria-label="info-circle"
        class="anticon anticon-info-circle ant-pro-core-label-tip-icon"
        role="img"
      >
        <svg
          aria-hidden="true"
          class=""
          data-icon="info-circle"
          fill="currentColor"
          focusable="false"
          height="1em"
          viewBox="64 64 896 896"
          width="1em"
        >
          <path
            d="M512 64C264.6 64 64 264.6 64 512s200.6 448 448 448 448-200.6 448-448S759.4 64 512 64zm0 820c-205.4 0-372-166.6-372-372s166.6-372 372-372 372 166.6 372 372-166.6 372-372 372z"
          />
          <path
            d="M464 336a48 48 0 1096 0 48 48 0 10-96 0zm72 112h-48c-4.4 0-8 3.6-8 8v272c0 4.4 3.6 8 8 8h48c4.4 0 8-3.6 8-8V456c0-4.4-3.6-8-8-8z"
          />
        </svg>
      </span>
    </div>
  </div>,
]
`;
